### DISCUSSION TITLE
Trouble downloading

### DISCUSSION THREAD

**Discussion by drydenem** [eclipse-birt/birt#896] [Q&A] [answered]
Hello! Super excited for the new release. I've been trying to download the new release on my 2020 macbook air with an M1 chip. Both all in one packages for macos give me the error "“eclipse” is damaged and can’t be opened. You should move it to the Trash." Any idea on how to get past this?

**Comment by wimjongman** [comment, marked as answer]
Yes. We have not yet had the chance to sign the mac builds. 

Mac users have to clear the quarantine attributes by executing the following command from the terminal window:

 xattr -c Eclipse.app

**Reply by drydenem** [reply]
Thank you! This worked!
